zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -3093,9 +3080,9 @@ components:
       type: object
       properties:
         filter:
-          allOf:
-            - $ref: "#/components/schemas/ProcessDefinitionFilterRequest"
+          $ref: "#/components/schemas/ProcessDefinitionFilterRequest"
```

Comment:
This is an example of my inheritance conversion. 

The left side would have required a `description` on the `filter` property; that description would never be emitted in the docs. The lack of it also caused vacuum to fail validation...with no message pointing to this line. 

The right side requires a description on the inherited `ProcessDefinitionFilterRequest` schema (and if it's not there, vacuum crashes 😅 😬 ). The description in the inherited schema does display in the docs.

Reply:
This makes a lot of sense and this is how it should be 👍 
The crashing vacuum is an issue, of course 😅